src/transforms/deprecation-warnings.ts

```diff
@@ -469,6 +469,9 @@ class Transformer {
   private getStatementsForDeclaration(
     node: ts.MethodDeclaration | ts.GetAccessorDeclaration | ts.SetAccessorDeclaration | ts.ConstructorDeclaration,
   ): ts.Statement[] {
+    const printer = ts.createPrinter();
+    console.log(printer.printNode(ts.EmitHint.Unspecified, node, node.getSourceFile()));
+
```

Comment:
Is this a left over debug print? if not - can you elaborate?

Reply:
Oops! Thanks for catching